README

```diff
@@ -42,6 +42,7 @@ CREATE TABLE users (
 	donated_slimes bigint NOT NULL DEFAULT '0',
 	donated_poudrins bigint NOT NULL DEFAULT '0',
 	arrested int NOT NULL DEFAULT '0',
+	splattered_slimes bigint NOT NULL DEFAULT '0',
```

Comment:
this column is declared but never used. shouldn't we change the loyalest consumers board to show this?